FBSDKShareKit/FBSDKShareKit/FBSDKShareAPI.m

```diff
@@ -204,6 +203,11 @@ - (BOOL)validateWithError:(NSError *__autoreleasing *)errorRef
 
 #pragma mark - Helper Methods
 
+- (FBSDKAccessToken *)_accessToken
```

Comment:
let's declare _accessToken as an ivar, and have the property lazily set it to `[FBSDKAccessToken currentAccessToken]`. That way, we can consistently use the `accessToken` property (i.e., prevent accidental usage of `accessTokenOverride` internally)